content/2017/02/06/server-side-rendering-client-side-app.rst

```diff
@@ -0,0 +1,257 @@
+public: yes
+author: jonny
+tags: [
+  'Single-Page Applications',
+  'Server-Side Rendering',
+  JavaScript,
+  Jinja2,
+  Django,
+  Python,
+  Code,
+  Backbone,
+  Marionette
+  ]
+image:
+  - src: 'http://placebear.com/1200/600'
+summary: |
+  We want the convenience of a client-side single-page application using our
+  `MV* framework`_ `of choice`_, but don't want to sacrifice the SEO and UX
+  benefits of rendering the initial page-load on the server. And while we're
+  not here to trash on Node.js, we're also not ready to commit to an
+  all-JavaScript tech stack. Here's our attempt to have the best of both
+  worlds, all the while keeping duplication of logic or code between the
+  front-end and back-end worlds to a minimum.
+
+  .. _MV* framework: http://backbonejs.org/
+  .. _of choice: http://marionettejs.com/
+
+
+Server-Side Rendering For Client-Side Apps
+==========================================
+
+Here at OddBird, we're lucky enough to mostly work on `greenfield projects`_ —
+which means we usually choose our own tech stack. One of the first things we
+deal with is how templates are rendered on initial page-load. There are many
+reasons to prefer server-side rendering over a "pure" single-page app which
+always renders content in the browser — it's better for SEO, users don't have
+to wait for all the JavaScript to initialize before seeing content on the page,
+etc. But it's also more work to convince a client-side `MV* framework`_ to play
+nicely (and efficiently!) with a server-rendered template.
+
+.. _greenfield projects: https://en.wikipedia.org/wiki/Greenfield_project
+.. _MV* framework: http://backbonejs.org/
```

Comment:
I'm fairly sure rst doesn't allow nesting links with bold/italic — so it just ignores that *